File: space-game/OrbitalMechanics/ISphere.vb
﻿Imports Core.Classes
Imports Newtonsoft.Json

Public Interface ISphere
    Inherits I3DObject

    ReadOnly Property Radius As Distance

    <JsonIgnore>
    ReadOnly Property AxialTilt As Integer

    <JsonIgnore>
    ReadOnly Property RotationSpeed As Integer

End Interface

File: space-game/OrbitalMechanics/Interfaces/ISphere.cs
using Core.Classes;
using Newtonsoft.Json;

namespace OrbitalMechanics.Interfaces
{
    public interface ISphere : I3DObject
    {
        Distance Radius { get; }
        [JsonIgnore]

        int AxialTilt { get; }
        [JsonIgnore]

        int RotationSpeed { get; }
    }
}
